scripts/deploy.sh

```diff
@@ -67,6 +70,7 @@ fi
 which openstack-ansible || ./scripts/bootstrap-ansible.sh
 
 # ensure all needed passwords and tokens are generated
+./scripts/pw-token-gen.py --file /etc/openstack_deploy/user_secrets.yml
```

Comment:
Yeah, I think this should be in the DEPLOY_AIO block, and I'm surprised that we need it, although it seems we may.